.storybook/webpack.config.js

```diff
@@ -0,0 +1,23 @@
+const path = require('path');
+
+// Export a function. Accept the base config as the only param.
+module.exports = async ({ config, mode }) => {
+  config.module.rules.push({
+    test: /.(js|jsx)$/,
+    exclude: /node_modules/,
+    use: ['babel-loader', 'eslint-loader'],
+},
+{
+    test: /.(jpg|jpeg|png|gif|svg)$/,
```

Comment:
great job @lemoissonM  among these extensions I highly recommend you to add `.webp` since we are going to use it largely in our frontend app